DashAI/back/main.py

```diff
@@ -18,6 +19,23 @@
 app.mount(settings.API_V0_STR, api_v0)
 app.mount(settings.API_V1_STR, api_v1)
 
+# Load registries
+task_registry = get_task_registry()
+model_registry = get_model_registry()
+
+registry_dict_plugins = {"model": model_registry, "task": task_registry}
```

Comment:
We shouldn't load these here. Also we should have a plugin_registry just like tasks and models.